<launch>

  <node pkg="lslidar_c32_driver" type="lslidar_c32_driver_node" name="leishen_lslidar_c32_driver" output="screen" >
    <param name="frame_id" value="ls"/>
    <param name="device_ip" value="192.168.1.200"/>
    <param name="device_port" value="2368"/>
  </node>

  <node pkg="lslidar_c32_decoder" type="lslidar_c32_decoder_node" name="leishen_lslidar_c32_decoder" output="screen">
    <param name="child_frame_id" value="ls"/>
    <param name="point_num" value="2000"/>
    <param name="channel_num" value="0"/>
    <param name="angle_disable_min" value="0"/>
    <param name="angle_disable_max" value="0"/>
    <param name="min_range" value="0.15"/>
    <param name="max_range" value="500.0"/>
    <param name="frequency" value="10.0"/>
    <param name="publish_point_cloud" value="true"/>
    <param name="publish_channels" value="false"/>
    <remap from="lslidar_point_cloud" to="/point_raw" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="aaa" args=" 0.0689343  0.675424 -0.180678 2.87252  -3.075 2.81061   /ls  /velodyne_right 10" />


  
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_c32_decoder)/launch/lslidar_c32.rviz" output="screen"/>

</launch>
